cumulus_library/studies/core/core_templates/diagnosticreport.sql.jinja

```diff
@@ -0,0 +1,102 @@
+{% import 'core_utils.jinja' as utils %}
+{% import 'unnest_utils.jinja' as unnest_utils %}
+
+-- This table includes all fields of interest to the US Core DiagnosticReport profiles.
+-- EXCEPT FOR:
+-- * the 'presentedForm' field, which is an attachment array that is stripped out by the ETL.
+-- * the `reporter` field, simply due to it not likely being interesting to consumers
+--   and being an array field, which would require a lot of row duplication.
+--
+-- US Core profiles for reference:
+-- * https://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-diagnosticreport-lab.html
+-- * https://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-diagnosticreport-note.html
```

Comment:
oh what a nice piece of documentation that we should replicate across all the main core templates